src/axom/spin/ImplicitGrid.hpp

```diff
@@ -346,6 +346,36 @@ class ImplicitGrid
     return res;
   }
 
+  /*!
+   * Finds the candidate elements in the given \a gridCell of the grid
+   *
+   * \param [in] gridCell The cell of the grid
+   * \return A bitset \a bSet whose bits correspond to the elements of the IndexSet.
```

Comment:
Is it appropriate to annotate `bSet` here since its not an argument?

Reply:
I think so (?)
I reworked the docs to remove it.